xapian-applications/omega/clickmodel/simplifieddbn.cc

```diff
@@ -73,9 +75,9 @@ getClicksList(vector<vector<string>> &sessions)
 		clickstring += sessions[i][CLICKS][j];
 
 		// Get click count string.
-		string delimiter = ":";
-		int delimiterPos = clickstring.find(delimiter);
-		clickcount = clickstring.substr(delimiterPos + 1, clickstring.length() - 1 - delimiterPos);
+		int delimiter_pos = clickstring.find(':');
+		clickcount = clickstring.substr(delimiter_pos + 1,
+			     clickstring.length() - 1 - delimiter_pos);
```

Comment:
There is `clickstring.length() - 1 - delimiter_pos` as the second argument of `substr()` here. Did I miss something else here?